src/MizarData/ActiveFunctionTimePerFrameComparatorTest.cpp

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <cstdint>
+#include <limits>
+#include <type_traits>
+#include <utility>
+
+#include "MizarData/ActiveFunctionTimePerFrameComparator.h"
+#include "MizarData/BaselineOrComparison.h"
+#include "MizarData/SampledFunctionId.h"
+#include "MizarData/SamplingWithFrameTrackComparisonReport.h"
+
+namespace orbit_mizar_data {
+
+using testing::DoubleNear;
+using testing::Return;
+
+namespace {
+class MockCounts {
+ public:
+  MOCK_METHOD(double, GetExclusiveRate, (SFID), (const));
+  MOCK_METHOD(uint64_t, GetTotalCallstacks, (), (const));
+};
+
+class MockFrameTrackStats {
+ public:
+  MOCK_METHOD(double, ComputeAverageTimeNs, (), (const));
+  MOCK_METHOD(uint64_t, variance_ns, (), (const));
+  MOCK_METHOD(uint64_t, count, (), (const));
+};
+
+}  // namespace
+
+constexpr uint64_t kTotalCallstacksBaseline = 100;
+constexpr uint64_t kTotalCallstacksComparison = 200;
+
+constexpr double kRateBaseline = 0.1;
+constexpr double kRateComparison = 0.15;
+
+constexpr uint64_t kFramesCntBaseline = 1000;
+constexpr uint64_t kFramesCntComparison = 1300;
```

Comment:
Don't abbreviate "count" to "cnt" (style guide: "Do not abbreviate by deleting letters within a word.")